iis-3rdparty-coansys-citation-matching/README.markdown

```diff
@@ -0,0 +1,11 @@
+# About
+
+All the scala, java and resource files in this module stored within the pl/edu/icm/ subdirectories were copied from the following CoAnSys repository locations:
+
+https://github.com/CeON/CoAnSys/tree/master/citation-matching/citation-matching-coansys-code
+
+https://github.com/CeON/CoAnSys/tree/master/ceon-scala-commons-lite
```

Comment:
I think we should document the exact commit `f0373f0416c93d3cff3a6ea137da09fba1481cad` from which the files were copied.